dcc.py

```diff
@@ -400,11 +468,22 @@ def dcc_main(apkfile, filtercfg, outapk, do_compile=True, project_dir=None, sour
         if not os.path.exists(project_dir):
             shutil.copytree('project', project_dir)
         write_compiled_methods(project_dir, compiled_methods)
+
+        if not dynamic_register:
```

Comment:
invert if condition to avoid negated boolean expression,positive boolean expressions are easier to understand.
```suggestion
        if dynamic_register:
```